src/sass/theme/_understrap.scss

```diff
@@ -170,3 +170,28 @@ a.skip-link {
 .gallery-caption {
 	display: block;
 }
+
+/* Search Widget */
+.wp-block-search {
+    .wp-block-search__label {
+        @extend .screen-reader-text;
+    }
+
+    .wp-block-search__inside-wrapper {
+        @extend .input-group;
+
+        .wp-block-search__input {
+            @extend .form-control;
```

Comment:
@bacoords a from-control mixin as mentioned in #1661 would make perfectly sense here.